lib/App/GitHooks.pm

```diff
@@ -837,6 +843,11 @@ sub get_config
 		);
 	}
 
+    # Enforce the specifying of min version of App::GitHooks
```

Comment:
Please switch to hard tabs in the entire diff, for consistency with the rest of the distribution.